optimum/utils/testing_utils.py

```diff
@@ -11,3 +14,41 @@ def require_hf_token(test_case):
         return unittest.skip("test requires hf token as `HF_AUTH_TOKEN` environment variable")(test_case)
     else:
         return test_case
+
+
+def require_sigopt_token_and_project(test_case):
+    """
+    Decorator marking a test that requires sigopt API token.
+    """
+    use_auth_token = os.environ.get("SIGOPT_API_TOKEN", None)
+    has_sigopt_project = os.environ.get("SIGOPT_PROJECT", None)
+    if use_auth_token or has_sigopt_project is None:
```

Comment:
Should it be `if use_auth_token is None or has_sigopt_project is None` ?

Reply:
Exactly! Will fix that!